catroid/src/org/catrobat/catroid/common/Constants.java

```diff
@@ -109,6 +109,11 @@
 	public static final int DOWNLOAD_NOTIFICATION = 103;
 	public static final int COPY_NOTIFICATION = 104;
 
+	//flavour (Pocket Code, Pocket Phiro,.....)
```

Comment:
right...I will shift them to the other web variables - because they need this information.
I will also delete the phiro-flavor variable -  we will need them at a later point ...